docs/installation.mdx

```diff
@@ -90,6 +95,18 @@ app, err := svixClient.Application.Create(&svix.ApplicationIn{
 })}
 ```
 
+</TabItem>
+<TabItem value="java">
+
+```java
+import com.svix.generated.model.ApplicationIn;
+import com.svix.generated.model.ApplicationOut;
```

Comment:
I'd like to hold off on merging this until we decide on https://github.com/svixhq/svix-libs/pull/49 as it changes these import paths.

Reply:
It's now merged.